.github/workflows/test.yaml

```diff
@@ -0,0 +1,379 @@
+name: test
+
+on:
+  push:
+    branches:
+      # - main
+      - chore/ci-for-example-notebooks
```

Comment:
Do you have to change this to main for the tests to get triggered ?

Reply:
yup!

Reply:
will change it back before merging